### ISSUE TITLE
OpenCOR 0.7 uses CPU even when idle

### ISSUE THREAD

**Issue by agarny** [opencor/opencor#2721]
When we fixed #2709, we introduced a regression whereby OpenCOR always uses some CPU (100% on my MacBook Pro!) even when idle. This is because toolbars keep on updating their style.

#2709 was about OpenCOR crashing after changing its style in the `Preferences` dialog. To change the style of OpenCOR was never a critical feature and was only (quickly) added because Qt allows to change the style of an application. However, toolbars in the version of Qt we use have some problems and we fixed them through styling, but this is causing other problems.

The bottom line is that there is no need for OpenCOR's style to be changeable. So, our simplest option to not to allow the user to change the style of OpenCOR.
